### ISSUE TITLE
add "wipe no data" as option in Data Wipe Setup

### ISSUE THREAD

**Issue by bxhghz** [SaferMobile/InTheClear#39]
in testing, users were confused about how to ensure that InTheClear does not wipe any data.  there should be an option in the list; ideally selecting it would automatically unselect any other options on the list.

**Comment by djhalliday** [comment]
yes, this exists in the Android version, so we should also add this to be conssitent!

**Comment by n8fr8** [comment]
Hmm... what about text that says "If you do not want to wipe data, then do not select any items below" ?

**Comment by bxhghz** [comment]
On 8/12/11 10:48 AM, n8fr8 wrote:

> Hmm... what about text that says "If you do not want to wipe data, then do not select any items below" ?
> 
> I think a box next to a "Wipe no data" option means the same thing, uses
> fewer words, and follows a pattern of selection that is common -- a sort
> of "none of the above" model.

## 

Becky Hurwitz
MobileActive.org
becky@mobileactive.org

skype: becky.hurwitz
twitter: @beckyhurwitz
+1 (347) 878-8239

Check out the new Mobile Media Toolkit!
MobileMediaToolkit.org Making Media Mobile

A global network of people using mobile technology for social impact
http://mobileactive.org

**Comment by n8fr8** [comment]
On 08/12/2011 11:19 AM, bxhz wrote:

> On 8/12/11 10:48 AM, n8fr8 wrote:
> 
> > Hmm... what about text that says "If you do not want to wipe data, then do not select any items below" ?
> > 
> > I think a box next to a "Wipe no data" option means the same thing, uses
> > fewer words, and follows a pattern of selection that is common -- a sort
> > of "none of the above" model.

Ah, yes, "None of the above" does make sense. There is a bit more logic
in the deselection of the other options, but I will see if we can get
this in. Otherwise, a no-brainer for 0.4.
